.travis.yml

```diff
@@ -0,0 +1,21 @@
+dist: xenial
+
+language: ruby
+
+git:
+  depth: 200
+
+rvm:
+  - 2.3.3
+
+before_install:
+  - sudo apt-get install -y libpango1.0-dev ghostscript
+  - gem install asciidoctor -v 1.5.8
+  - gem install coderay json-schema
+  - gem install ttfunk -v 1.5.1
+  - gem install asciidoctor-pdf -v 1.5.0.beta.8 --pre
```

Comment:
asciidoctor-pdf 1.5.0 does exit now. Should we move to that instead of using a beta version?

Reply:
We will need to use version 1.5.0.beta.8 at least until #215 is merged, due to an issue with the asciidoctor-pdf-monkeypatch script.  If we merge #215 first, I can look at switching to 1.5.0 as part of this PR.

Reply:
1.5.0 comes after 1.5.0.beta.8 ;)